lib/content-services/src/lib/content.module.ts

```diff
@@ -82,7 +83,8 @@ import { ContentUserInfoModule } from './content-user-info/content-user-info.mod
         NodeCommentsModule,
         TreeModule,
         SearchTextModule,
-        AlfrescoViewerModule
+        AlfrescoViewerModule,
+        AdminCcSecurityControlsServicesModule
```

Comment:
can we please drop AdminCC prefix from the service and Ng module? so that it's just "SecurityControlsServiceModule"

Reply:
Renamed, ty